modules/8_reinforcement_learning/reinforcement_learning.py

```diff
@@ -0,0 +1,128 @@
+import itertools
+import os
+
+import numpy as np
+from gym import wrappers
+from gym.envs import toy_text
+
+import test_reinforcement_learning
+
+
+WEIGHTS_PATH = os.path.join(os.path.dirname(__file__), 'q_values.npy')
```

Comment:
Please search for this file in `data` folder. See https://github.com/dkurt/openvino_practice/blob/develop/modules/7_colorization/test_colorization.py

Reply:
Changed